Eastern-Research-Group/lew
# #99 Upgraded the node engine to the latest.

## Related Issues:
* #99

## Main Changes:
* Upgraded the engines node version to the latest in the package.json.
* Upgraded the CircleCI image to the latest node version.

## Steps To Test:
1. Run `npm run test` and verify the tests pass
2. Start the app
3. Verify the features still work
4. Merge this in 
5. Verify CircleCI logs look good
6. Verify the app works on the dev site
